server/controllers/report.js

```diff
@@ -870,18 +870,50 @@ function donation_confirmationRecords(params) {
       _end =  sanitize.escape(util.toMysqlDate(new Date(p.end)));
 
   var sql =
-    'SELECT `donations`.`uuid`, `donations`.`date`, `user`.`first`, `user`.`last`, `employee`.`prenom`, ' +
-    '`employee`.`name`, `employee`.`postnom`, `donor`.`name` AS `donor_name` ' +
+    'SELECT DISTINCT `donations`.`uuid`, `donations`.`date`, `user`.`first`, `user`.`last`, `employee`.`prenom`, ' +
+    '`employee`.`name`, `employee`.`postnom`, `donor`.`name` AS `donor_name`, ' +
+    '`posting_journal`.`trans_id` AS `journal_trans_id`, `general_ledger`.`trans_id` AS `ledger_trans_id` ' +
     'FROM `donations` ' +
     'JOIN `user` ON `user`.`id` = `donations`.`confirmed_by` ' +
     'JOIN `donor` ON `donor`.`id` = `donations`.`donor_id` ' +
     'JOIN `employee` ON `employee`.`id` = `donations`.`employee_id` ' +
+    'LEFT JOIN `posting_journal` ON `posting_journal`.`inv_po_id` = `donations`.`uuid` ' +
+    'LEFT JOIN `general_ledger` ON `general_ledger`.`inv_po_id` = `donations`.`uuid`' +
     'WHERE DATE(`donations`.`date`) BETWEEN DATE(' + _start + ') AND DATE(' + _end + ') ' +
     'AND `donations`.`is_confirmed` = 1 ORDER BY `donations`.`date` DESC ;';
 
   return db.exec(sql);
 }
 
+function integration_stock(params) {
+  var p = querystring.parse(params);
+
+  var _start = sanitize.escape(util.toMysqlDate(new Date(p.start))),
+      _end =  sanitize.escape(util.toMysqlDate(new Date(p.end))),
+      _depot = sanitize.escape(p.depot);
+
+  var sql =
```

Comment:
In ECMAScript 6 (the next version of JavaScript), the `\`` tick marks are used for [string templating](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings).  I know we have been using them in our SQL strings for a long time, but let's start avoiding them.  If you could remove them here, it would be a good start.